Adyen/UI/Views/NetworkImageView.swift

```diff
@@ -34,10 +34,26 @@ open class NetworkImageView: UIImageView {
     }
     
     // MARK: - Private
+
+    private let brandProtectedImages: [String: String] = ["applepay": "brand_apple_pay"]
     
     private var dataTask: URLSessionDataTask?
     
+    private func setImage(_ image: UIImage) {
+        DispatchQueue.main.async {
+            self.image = image
+            self.dataTask = nil
+        }
+    }
+
     private func loadImage(from url: URL) {
+        let parts = url.lastPathComponent.split(separator: "@")
+        if parts.count > 1,
+           let internalName = brandProtectedImages[String(parts[0])],
+           let image = UIImage(named: internalName, in: Bundle.coreInternalResources, compatibleWith: nil) {
+            return setImage(image)
+        }
```

Comment:
why use a local image? this way we need to update the SDK whenever we want to update apple icon, why not update the image on the backend?